woocommerce/woocommerce
# Fix WP latest-2 version retrieval in the "Smoke test release" workflow.

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Since the release of WP 6.2, I discovered that the "Smoke test release" workflow does not correctly get the WP L-2 correctly because of a missing logic somewhere in this workflow. The workflow is incorrectly getting 6.1.0 as the L-2 version as shown [here](https://github.com/woocommerce/woocommerce/actions/runs/4633367412/jobs/8198513829#step:6:27), while it's supposed to get `6.0.3`.

This PR fixes the way the L-2 version is retrieved. 

### How to test the changes in this Pull Request:

1. Go to the [Smoke test release](https://github.com/woocommerce/woocommerce/actions/workflows/smoke-test-release.yml) workflow page.
2. Select this branch (`fix/wp-l2-retrieval`) and run the workflow.
3. Open the workflow run.
4. In the `Get WP L-1 & L-2 version numbers` job, expand the `Get version numbers` step.
5. Verify that the L-1 and L-2 versions shown in the logs are `6.1.1` and `6.0.3`, respectively. 
    <img width="274" alt="LkhcF9mQ8G" src="https://user-images.githubusercontent.com/4509348/232121901-93db6868-6014-4822-b388-2d20257bcd43.png">
